### DISCUSSION TITLE
ANN: Projects now have a homepage

### DISCUSSION THREAD

**Discussion by williamstein** [sagemathinc/cocalc#6407] [Announcements]
This is still very, very minimal, but CoCalc projects now have a "Home page":

<img width="900" alt="image" src="https://user-images.githubusercontent.com/1276278/217127751-76429c2a-4347-4e75-82ff-cfc41322cfe0.png">

When you click on the new Home button in the upper left of a project (above the management pages and to the left of file tabs), it shows the project homepage.  Right now this just shows the project title, the project "avatar" image, and to the right the project description rendered as markdown.    Also, the log of recent project activity is displayed below.

This page will get more intelligent and interesting (and configurable) as we develop it further.

**Comment by StevenClontz** [comment]
Just came here to ask about this - right now I keep clicking it when I'm trying to get to my file explorer, and it's very unclear why this mostly useless page is taking up this real estate (unclear so far - I'm hopeful you've got some cool ideas for it down the line).

**Comment by williamstein** [comment]
>  (unclear so far - I'm hopeful you've got some cool ideas for it down the line).

The cool idea, which shouldn't be too far in the future, is that you'll be able to drag around components and place them however you want on this page.  The components will include the explorer, log, and other vertical content.  So hopefully soon you could just put a big explorer here and then your problem will be solved.  But, I can see how it's just annoying right now.

**Comment by DrXyzzy** [comment]
CoCalc online docs are updated to include this feature: https://doc.cocalc.com/project-home.html
